demisto_sdk/commands/common/errors.py

```diff
@@ -1558,6 +1561,22 @@ def taskid_different_from_id(task_key, id_, taskid):
         return f"On task: {task_key},  the field 'taskid': {taskid} and the 'id' under the 'task' field: {id_}, " \
                f"must be with equal value. "
 
+    @staticmethod
+    @error_code_decorator
+    def playbook_not_quiet_mode():
```

Comment:
if one should be able to ignore those errors, they should be added to `ALLOWED_IGNORE_ERRORS`